sentry_docker_conf.py

```diff
@@ -98,6 +98,9 @@ def nydus_config(from_env_var):
 # You MUST configure the absolute URI root for Sentry:
 SENTRY_URL_PREFIX = config('SENTRY_URL_PREFIX')  # No trailing slash!
 
+SCRIPT_NAME = config('SCRIPT_NAME', '')
```

Comment:
I don't think this is necessary. There's no setting named ``SCRIPT_NAME`` neither in Django nor in Sentry.